src/fragmentarium/ui/info/Details.tsx

```diff
@@ -73,7 +74,17 @@ function Accession({ fragment }: Props) {
   return <>Accession: {fragment.accession || '-'}</>
 }
 
-function Details({ fragment }: Props) {
+type DetailsProps = {
+  fragment: Fragment
+  updateGenres: (genres: Genre[]) => any
```

Comment:
Do not use `any`. If you do not know or care about the result, use `unknown`.

Currently, this will cause a warning and it would be good if the new code does not add any new warnings. I am going to change it to an error in the future, but currently, there is too much un-typed code.